### DISCUSSION TITLE
how to add dependency of docker-java-transport-httpclient5

### DISCUSSION THREAD

**Discussion by champagne** [docker-java/docker-java#1910] [Q&A] [answered]
I tried below, but got failure.
		<dependency>
			<groupId>com.github.docker-java</groupId>
			<artifactId>docker-java-transport-httpclient5</artifactId>
			<version>3.2.13</version>
		</dependency>
Checked my local .m2/repository, I found there's no docker-java-transport-httpclient5.
![image](https://user-images.githubusercontent.com/82297/181204346-ce669554-d919-404d-8579-ec6b0bfa0dda.png)

**Comment by champagne** [comment]
I don't want to use docker-java-transport-jersey or docker-java-transport-netty.

**Comment by champagne** [comment, marked as answer]
Solved by below, not sure why not available before.
```
		<dependency>
		   <groupId>com.github.docker-java</groupId>
		   <artifactId>docker-java-transport-httpclient5</artifactId>
		   <version>3.2.13</version>
		</dependency>
```

**Reply by eddumelendez** [reply]
that's great! yeah, it should be available https://mvnrepository.com/artifact/com.github.docker-java/docker-java-transport-httpclient5/3.2.13
